src/cython/vapoursynth.pyx

```diff
@@ -2496,6 +2498,15 @@ cdef class Plugin(object):
             func = self.funcs.getNextPluginFunction(func, self.plugin)
             yield tmp
 
+    @property
+    def __version__(self):
```

Comment:
Remove the dunder.

Reply:
Reason i added the dunder is some packages actually use version/Version as name for functions, and i didn't want to break those.